### DISCUSSION TITLE
Running 2 separate Airbyte docker instances on one server

### DISCUSSION THREAD

**Discussion by reallywhoami** [airbytehq/airbyte#32809] [Connector Questions]
Hello, I am trying to run 2 Airbyte instances using docker on one server. The problem is that when I'm changing container names, because of the second instance they need to be different, some internal code of the application stops working and 2 containers keep restarting: airbyte-proxy and airbyte-webapp. In logs I get error: Host not found in upstream “airbyte-server:8001” in /etc/nginx/conf.d/default.conf". Is there any way to manage 2 Airbyte instances with 2 docker-compose files?
https://discuss.airbyte.io/t/host-not-found-in-upstream-airbyte-server-8001-in-etc-nginx-conf-d-default-conf/3203/6
https://github.com/airbytehq/airbyte/issues/19787#issuecomment-1328593860

**Comment by marcosmarxm** [comment]
I don't think it is possible to have in Docker deployment without changing some code.

**Reply by reallywhoami** [reply]
So I can't have more than one Airbyte instance?

**Reply by marcosmarxm** [reply]
Using default deployment mode, no it isn't possible.

[a comment by an automated account is not shown]
